### DISCUSSION TITLE
new mruby.org web.

### DISCUSSION THREAD

**Discussion by Hir0** [mruby/mruby.github.io#148] [General]
Hi, everyone.

Below is a design demo of the new mruby.org web.

https://comfy-bonbon-af03a5.netlify.app/

The mruby.org homepage is already over 10 years old and we wanted to add more information.
We have added a new get started page for beginners and how to use ruby with microcontrollers such as M5Stack and micro: bit.
We would like to hear your various opinions.
   ＊Some of the topics are dummy text.
Thank you.

Hiromasa

**Comment by bovi** [comment]
I like it 🥰

**Reply by Hir0** [reply]
Thanks :)

**Comment by jbampton** [comment]
Wow looks awesome @Hir0 🥇 

Love the new modern design it is much better for marketing.
